recipes-graphics/wayland/weston_8.0.0.imx.bb

```diff
@@ -144,7 +144,7 @@ PACKAGECONFIG_OPENGL              = "opengl"
 PACKAGECONFIG_OPENGL_imxgpu2d     = ""
 PACKAGECONFIG_OPENGL_imxgpu3d     = "opengl"
 
-PACKAGECONFIG_remove = "wayland x11"
+PACKAGECONFIG_remove = "wayland x11 xwayland"
```

Comment:
Does it fix a build error? or it is just for consistency?

Reply:
This actually fixes a build error, it has been reported on Yocto IRC. Original description is like that:

>  I'm trying to build the fsl-image-multimedia-full recipe, which relies on weston-8.0.0.imx.bb. That recipe is failing to build with an error about "ERROR: Problem encountered: xwayland requires xcursor which was not found. Or, you can use '-Dxwayland=false'." But I have xcursor, and xcursor-devel installed on my build system, and xcursor recipes exist within the includd meta-layers. Can anyone point me in the right direction here?

It turned out that when both X11 and wayland were disabled - xwayland cannot be enabled.